vignettes/write-to-disk.Rmd

```diff
@@ -0,0 +1,39 @@
+---
+title: "Mocking writing to disk"
+author: "Scott Chamberlain"
+date: "`r Sys.Date()`"
+output:
+  html_document:
+    toc: true
+    toc_float: true
+    theme: readable
+vignette: >
+  %\VignetteIndexEntry{Writing to disk}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r echo=FALSE}
+knitr::opts_chunk$set(
+    comment = "#>",
+    collapse = TRUE,
+    warning = FALSE,
+    message = FALSE
+)
+```
+
+Request matching
+================
+
+```{r setup}
+library("vcr")
+vcr::turn_on()
```

Comment:
Actually it's not very good to have this line here. I think it is because the lightswitch vignette turns off vcr. Could the lightswitch vignette "clean after itself"?

Reply:
What do you suggest?

Reply:
not sure, maybe adding a vcr::turn_on at the end of the lightswitch vignette. What confuses me is: aren't vignette knit in a clean environment?

Reply:
Not sure about the clean env thing.  Not sure what exactly is wrong here. The vignettes build just fine right?

Reply:
Without the vcr::turn_on() above there was an error saying vcr was not enabled.

Reply:
Okay, probably related to previous vignette settings. Can we just put it in a different block and not show it (echo=FALSE or whatever it is)

Reply:
any thoughts @maelle ?

Reply:
I'll experiment in a minimal package next week, I want to understand more about how a vignette can influence another vignette.

Reply:
yes, there's no new content after all, only doc infrastructure change.